backend/src/models/user.model.ts

```diff
@@ -11,12 +11,13 @@ export interface IUser extends Document {
   image?: string;
   socials?: [{ link: string; social: ISocial['_id'] }];
   active: boolean;
+  section: boolean;
 }
 
 const userSchema = new Schema({
   name: String,
   title: String,
-  email: { type: String, required: true, unique: true },
+  email: { type: String, unique: true },
```

Comment:
I think if you changed that an email is not required You need to update the validator also.